web/src/stores/workflow-store.ts

```diff
@@ -106,6 +112,7 @@ export const useWorkflowStore = create<WorkflowStoreState>((set, get) => ({
 	isActive: false,
 	nodes: [],
 	edges: [],
+	missingSecretForNodes: undefined,
```

Comment:
I think this name is confusing because we are tracking nodes with deleted secrets.